frontend/src/components/SelectedPlaylistSongs.tsx

```diff
@@ -66,11 +67,16 @@ export const SelectedPlaylistSongs = ({ playlistLink }: Props): JSX.Element => {
     songIdList();
   }, [PLAYLIST_LINK, playlistsRef]);
 
-  return isLoading ? (
-    <div>Loading...</div>
-  ) : songLinksRef.current ? (
-    <div>{songLinksRef.current.toString()}</div>
-  ) : (
-    <div>No songs to display</div>
-  );
-};
+// return  (<SelectedPlaylistSongsAnalysis playlistLink={songLinksRef.current})
+
+return (
+  <div>
+    {isLoading ? (
+      <h1>Loading...</h1>
+    ) : (
+      <SelectedPlaylistSongsAnalysis TrackIDList={songLinksRef.current} />
+    )}
+  </div>
+);
```

Comment:
```suggestion
if (isLoading) {
  return (
    <h1>Loading...</h1>
  )
}
return (
  <div>
      <SelectedPlaylistSongsAnalysis TrackIDList={songLinksRef.current} />
  </div>
);
```

ternarys often make the code harder to read even if it seems like its less lines of code or more clever. in general we should prefer to be overly explicit with our code to improve readability. anotther downside of ternarys is that they become hard to maintain when the return values get more complex